Dockerfile-ModelBuilder

```diff
@@ -18,24 +19,35 @@ RUN cat /code/requirements.txt | grep tensorflow== > /code/prereq.txt \
 
 FROM python:3.6.9-slim-stretch
 
+# Nonroot user for running CMD
+RUN groupadd -g 999 gordo && \
+    useradd -r -u 999 -g gordo gordo
+
 # Install requirements separately for improved docker caching
-COPY --from=builder /code/prereq.txt .
-RUN pip install --no-deps -r prereq.txt --no-cache-dir
+COPY --from=builder /code/prereq.txt /home/gordo/
+COPY --from=builder /code/dist/gordo-components-packed.tar.gz /home/gordo/
+COPY requirements.txt /home/gordo/
 
-COPY requirements.txt /code/
-RUN pip install -r /code/requirements.txt --no-cache-dir
+ADD build.sh /home/gordo/build.sh
 
+# build.sh (build the model) as executable default command
+RUN cp /home/gordo/build.sh /usr/bin/build \
+    && chmod a+x /usr/bin/build
 
-# Install gordo-components, packaged from earlier 'python setup.py sdist'
-COPY --from=builder /code/dist/gordo-components-packed.tar.gz .
+# Make gordo own all in its home
+RUN chown -R gordo:gordo /home/gordo
 
-# Install gordo-components, packaged from earlier 'python setup.py sdist'
-RUN pip install ./gordo-components-packed.tar.gz
+# Switch user and update needed env vars
+USER gordo
+ENV HOME "/home/gordo"
+ENV PATH "${HOME}/.local/bin:${PATH}"
 
-ADD build.sh /code/build.sh
+# Install requirements and gordo-components packaged in intermediate image 'python setup.py sdist'
+RUN pip install --no-deps -r /home/gordo/prereq.txt --no-cache-dir --user
```

Comment:
We want this step before we even copy over `code/dist/gordo-components-packed.tar.gz`, so we better utilize docker caching.

Reply:
By *this step* you mean only the `prereq.txt`, right?